openpnm/algorithms/_transient_multiphysics.py

```diff
@@ -0,0 +1,121 @@
+import numpy as np
+from openpnm.utils import logging, SettingsAttr, Docorator
+from openpnm.integrators import ScipyRK45
+from openpnm.algorithms import GenericAlgorithm
+logger = logging.getLogger(__name__)
+docstr = Docorator()
+
+
+@docstr.dedent
+class TransientMultiPhysicsSettings:
+    r"""
+
+    Parameters
+    ----------
+    %(GenericAlgorithmSettings.parameters)s
+    algorithms: list
+        List of transient algorithm objects to be solved in a coupled manner
+
+    """
+    algorithms = []
+
```

Comment:
You need another blank like here. Basically, PEP8 suggests two blank lines between methods/classes in a module